artistools/lightcurve.py

```diff
@@ -281,6 +292,7 @@ def colour_evolution_plot(filter_name1, filter_name2, modelpath, args):
     directory = os.getcwd().split('/')[-2:]
     plt.legend(loc='best', frameon=True)
     plt.title(directory)
+    plt.title('B-V Colour Evolution')
```

Comment:
I'm not sure what happens if you have one plt.title after another. I'm assuming it just replaces what was already there, making the plt.title(directory) line redundant. I could be wrong though!